test/pummel/test-vm-memleak.js

```diff
@@ -19,12 +19,17 @@
 // OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+// Flags: --max_old_space_size=32
+
 var assert = require('assert');
 var common = require('../common');
 
 var start = Date.now();
 var maxMem = 0;
 
+var ok = process.execArgv.some(''.includes.bind('--max_old_space_size=32'));
```

Comment:
<3 new ES6 string stuff too, but this is really obtuse.
What's with good ol' indexOf?